photo_points/Repositories/FakeAdminReviewRepository.cs

```diff
@@ -56,8 +69,9 @@ public Capture GetCapture(long id)
 
         public IEnumerable<Capture> GetAllUnapproved()
         {
-            return captures.Where(a => a.Approved == true);
-            //Needs Willie's updated Capture model with approved property
+           // return captures.Where(a => a.approve == true);
+         return captures.Where(a => a.approval == Capture.ApprovalType.Approve);
```

Comment:
This seems to be returning all of the Approved photos rather than the "Unapproved"

Reply:
that is Eric codes i just  updated Name

Reply:
i will change it to pending